### ISSUE TITLE
Use proper namespace

### ISSUE THREAD

**Issue by voxik** [fog/fog-brightbox#46]
fog-brightbox is not compatible with recent fog-core and the test suite fails:

~~~
  Fog::Compute[:brightbox] (brightbox)
    brightbox has no compute service (Fog::Service::NotFound)
      /usr/share/gems/gems/fog-core-2.1.2/lib/fog/core/services_mixin.rb:18:in `rescue in new'
      /usr/share/gems/gems/fog-core-2.1.2/lib/fog/core/services_mixin.rb:7:in `new'
      /usr/share/gems/gems/fog-core-2.1.2/lib/fog/compute.rb:32:in `new'
      /usr/share/gems/gems/fog-core-2.1.2/lib/fog/core/services_mixin.rb:4:in `[]'
      tests/brightbox/compute_tests.rb:2:in `block in <top (required)>'
      /usr/share/gems/gems/shindo-0.3.8/lib/shindo.rb:79:in `instance_eval'
      /usr/share/gems/gems/shindo-0.3.8/lib/shindo.rb:79:in `tests'
      /usr/share/gems/gems/shindo-0.3.8/lib/shindo.rb:38:in `initialize'
      /usr/share/gems/gems/shindo-0.3.8/lib/shindo.rb:13:in `new'
      /usr/share/gems/gems/shindo-0.3.8/lib/shindo.rb:13:in `tests'
      tests/brightbox/compute_tests.rb:1:in `<top (required)>'
      /usr/share/gems/gems/shindo-0.3.8/lib/shindo/bin.rb:61:in `load'
      /usr/share/gems/gems/shindo-0.3.8/lib/shindo/bin.rb:61:in `block (2 levels) in run_in_thread'
      /usr/share/gems/gems/shindo-0.3.8/lib/shindo/bin.rb:58:in `each'
      /usr/share/gems/gems/shindo-0.3.8/lib/shindo/bin.rb:58:in `block in run_in_thread'
~~~

This could be probably fixed by correct namespacing, e.g. Fog::Brightbox::Compute instead of Fog::Compute::Brightbox. This is also suggested by fog/fog-core/pull/229

**Comment by tokengeek** [comment]
I believe this is covered on the https://github.com/fog/fog-brightbox/tree/version_1_release branch.

I see if we can finalise testing next week and release the changes.

**Comment by voxik** [comment]
There are just 5 commits and none of them is changing namespace, so I am skeptical

**Comment by tokengeek** [comment]
My mistake, it was https://github.com/fog/fog-brightbox/tree/version_1_rc2 which I hadn't pushed since the testing is not complete.

**Comment by voxik** [comment]
https://github.com/fog/fog-brightbox/commit/bf49d73d9ad866b19952cf9d8f4ac9ba8db0a854 indeed fixes this issue. Thx.

**Comment by tokengeek** [comment]
@voxik FYI - I have finished testing and found a regression in the code above when trying to destroy models.

That has been fixed and a final `fog-brightbox v1.0.0` has been released and is now available on rubygems.

Since this is now no longer a pre-release gem it should also be being picked up by the main `fog` gem

**Comment by voxik** [comment]
I put everything together and it seems it works. Thx.
